com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/security/data/termination/DataTerminationService.java

```diff
@@ -53,7 +53,7 @@
     private FileService fileService;
 
     @Inject
-    private ApplicationConfiguration configuration;
+    private ApplicationConfiguration applicationConfiguration;

```

Comment:
maybe a shorter name? appConfig?

Reply:
Decide to name it configuration and change at line 113 "configuration" variable to configurationEntity